### DISCUSSION TITLE
selectBooleanCheckbox hide/show panel not working

### DISCUSSION THREAD

**Discussion by prancius** [primefaces/community#3201] [PrimeFaces]
<code>
 <h:body>

        <h:form>

            <p:selectBooleanCheckbox label="Deklaruoti" value="#{everyPayBean.deklaruoti}">
                <p:ajax listener="#{everyPayBean.kazkas}" update="aaa bbb"/>
            </p:selectBooleanCheckbox>
          
            <p:commandButton id="bbb" widgetVar="bbb" disabled="#{everyPayBean.deklaruoti}" value="test"></p:commandButton>
            <p:panel id="aaa" widgetVar="aaa" rendered="#{everyPayBean.deklaruoti}">
                <p:outputLabel value="LLL"></p:outputLabel>
            </p:panel>
        </h:form>
    </h:body>
</code>

Any ideas why penel not showing on checkBox event?

If using 
update="@form" in ajax it works ..., but need hide/show only one panel and do not update whole form.

**Comment by tandraschko** [comment]
because you cant update a non-rendered element, as its not there to update in the view currently. you need to update a parent container.
